_sections/90-apps.md

```diff
@@ -1,8 +0,0 @@
----
-title: Apps
```

Comment:
It didn't seem right to me. I'm afraid "we" end up having to assess whether an app is compliant enough to make it to this list (and then keep doing that while the app evolves). I think it's better for tools themselve (such as Enketo) to advertise that they are following the spec. There is also the question of whether proprietary tools should be added and if not, what are acceptable licenses.